### ISSUE TITLE
Clarification question about minimal EGFR model

### ISSUE THREAD

**Issue by jadeshi** [sorgerlab/rasmodel#12]
Hi, I had a question about the minimal EGFR model shown in the tutorial:
http://rasmodel.org/components/egfr/egfr_minimal.html

Specifically, I'd like to ask about the interplay between the two rules of EGFR dimerization and EGFR/GRB2 binding. The EGFR dimerization rule specifies that two monomer EGFR's combine into a complexed EGFR dimer:

Rule(u'EGFR_EGF_EGFR_EGF_bind', EGFR(egf=ANY, egfr=None) +
        EGFR(egf=ANY, egfr=None) >> EGFR(egf=ANY, egfr=1) % EGFR(egf=ANY, egfr=1), kf_ee_bind_2)

Then, later on there is a rule that states that GRB2 binds to EGFR to form its own complex:

Rule(u'GRB2_EGFR_bind', GRB2(egfr=None) + EGFR(grb2=None) >>
        GRB2(egfr=1) % EGFR(grb2=1), kf_ge_bind_1)

However, as I understand it, dimerization of EGFR is a prerequisite to binding of GRB2, but with these two rules it seems like it's treating the dimerization of EGFR and binding of GRB2 as two competing pathways. Is this the case, or am I misinterpreting these rules? Thanks!

**Comment by bgyori** [comment]
You are right, the sentence "The adaptor protein GRB2 can bind EGFR that is phosphorylated on tyrosine. " should have produced a rule where GRB2 only binds EGFR if the latter is phosphorylated. It looks like, when we ran the reading/assembly (a very very long time ago), the sentence interpretation missed the phosphorylation condition.
